yellowbrick/classifier/classification_report.py

```diff
@@ -188,6 +190,13 @@ def score(self, X, y):
         y_pred = self.predict(X)
         scores = precision_recall_fscore_support(y, y_pred)
 
+        # Decode the target with the label encoder and get human readable labels
```

Comment:
@pkaf Are lines 193 to 199 necessary?  Can you explain your logic here?  

@bbengfort Did I miss something here?